compiler/src/iree/compiler/Dialect/Flow/IR/FlowOpFolders.cpp

```diff
@@ -1221,6 +1221,10 @@ static ElementsAttr tensorSlice(ElementsAttr tensor, uint64_t dim,
 
 OpFoldResult TensorSliceOp::fold(FoldAdaptor operands) {
   if (llvm::count(operands.getOperands(), nullptr) == 0) {
+    // Ignore DenseResources for now and do not perfom folding on them.
```

Comment:
I think this is totally valid to ignore. The elements might be stored in some file